lua/bufferline/state.lua

```diff
@@ -40,6 +40,7 @@ local utils = require'bufferline.utils'
 --- @field pins {[integer]: boolean} whether a buffer is pinned
 local state = {
   is_picking_buffer = false,
+  loading_session = false,
```

Comment:
Only one request, can you add an `@field` entry for `loading_session` under the `@class` comment just above here? With the other `@field`s. This should be good:

```lua
--- @field loading_session boolean `true` if a `SessionLoadPost` event is being processed
```

Reply:
Done